### DISCUSSION TITLE
Togglable skin exit and welcome sounds.

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#17140] [Ideas] [answered]
I noticed that lazer does not use the Welcome or exit sounds from my skin. While I don't mind not hearing my welcome sound as it's pretty basic I really miss my seeya.ogg sound as it's a goofy one I added specifically cause even if I'm having a crap day on osu and I rage quit I still Giggle everytime I hear my exit sound. Would be nice if this feature was toggleable so you could either use the one in your skin or use lazer's sounds.
Can't send my exit sound like I wanted cause .ogg is unsupported rip.

**Comment by frenzibyte** [comment, marked as answer]
Will be resolved by #17196

**Reply by Naozumi520** [reply]
#17196 (2022.314.0) was released and it looks great, thanks!
But in settings I see that there's no option in "Intro sequence" to let user to choose use the skin intro or default.

**Reply by peppy** [reply]
This will eventually be part of the skin editor. Not a game setting.
